compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELowerExecutableTarget.cpp

```diff
@@ -93,11 +140,24 @@ void AMDAIELowerExecutableTargetPass::runOnOperation() {
   if (translationInfo.has_value()) {
     switch (translationInfo.value().getDispatchLoweringPassPipeline()) {
         // Transform-dialect pipelines.
-      case IREE::Codegen::DispatchLoweringPassPipeline::TransformDialectCodegen:
+      case IREE::Codegen::DispatchLoweringPassPipeline::
+          TransformDialectCodegen: {
         addTransformDialectPasses(executableLoweringPipeline);
         break;
+      }
+      // TODO(avarma): Currently we are using "CPUDoubleTilingExpert".
+      //               Should we have "AIEDoubleTilingExpert" instead? This
+      //               seems to either be addressed in IREE repo or somehow we
+      //               are to extend "DispatchLoweringPassPipelineEnum" defined
+      //               inside IREECodegenAttrs.td.
+      case IREE::Codegen::DispatchLoweringPassPipeline::CPUDoubleTilingExpert: {
+        TilingConfig tilingConfig = getTilingConfigForPipeline(moduleOp);
+        addMultiTilingExpertPassPipeline(executableLoweringPipeline,
+                                         tilingConfig);
+        break;
+      }
```

Comment:
Extending on the above CPU vs AIE vs Common discussion.

What should be our modus operandi here?

To me, it seems like we need to add the corresponding enums in IREECodegenAttrs.td.
Besides the general `CPU` -> `AIE` renaming, I believe this would be required to also tackle the PAD vs PACK pipeline approach in AIE.